scripts/west_commands/runners/bossac.py

```diff
@@ -98,6 +98,7 @@ def get_offset(self, supports_offset):
                 ' Please see' +
                 ' https://github.com/zephyrproject-rtos/sdk-ng/issues/234' +
                 ' which tracks updating the Zephyr SDK.')
+            return None
```

Comment:
This belongs in a separate commit.

Reply:
Ah apologies, unintentional leftover from my private test setup.